code/core/src/manager/components/sidebar/SidebarBottom.tsx

```diff
@@ -164,7 +164,7 @@ export const SidebarBottomBase = ({ api, notifications = [], status = {} }: Side
   );
 
   useEffect(() => {
-    const wrapper = document.getElementById('sidebar-bottom');
+    const wrapper = document.getElementById('sidebar-bottom-wrapper');
```

Comment:
would be nice to have this in a constant so we don't miss things if we ever end up changing it. Plus add a comment so people know that this ID is important to stay consistent